lib/Twig/Deprecations.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class Twig_Deprecations
```

Comment:
What about naming it ``Twig_Util_DeprecationsCollector``, or ``Twig_Tool_DeprecationsCollector`` ? We could then add more utility tools in this "namespace", for instance extracting the linting logic out of the command of the Symfony bridge to make it available to anyone.